pegasus/helpers/asset_helpers.rb

```diff
@@ -4,13 +4,18 @@
 class AssetMap
   include Singleton
```

Comment:
Given that this is a singleton, how is `initialize` being called more than once? Is the issue that `minifiable_asset_path` or similar is being called by one thread before `initialize` finishes in another thread?